Back-End-Node/server.js

```diff
@@ -30,6 +30,7 @@ const redirect_uri = `${process.env.BACK_END_URL_SLACK}/auth/redirect`;
 const client = new WebClient();
 
 const createToken = (userId, role) => {
+  // TODO: it is confusing at the moment that we are using Slack title as the role here
```

Comment:
I explained this a few times at the previous meetings and presentations. We don't have "about me" page, I tried to add it but it is premium custom. Because of this there is only place that we can get role is  from "title".

Reply:
First commit was about "why we are getting role from title". 
Also our app is cretaing a token only normal user. We don't use verifyAdmin. Because we don't want any admin user with sign up. There is only way to assign admin is from database. I asked this how we need to assing admin. But i did not get eny sugestion. I found this way. If you have any, I am happy to learn and impliment it.